cpu/stm32/periph/rtt_f1.c

```diff
@@ -189,11 +192,19 @@ void RTT_ISR(void)
 {
     if (RTT_DEV->CRL & RTC_CRL_ALRF) {
         RTT_DEV->CRL &= ~(RTC_CRL_ALRF);
-        alarm_cb(alarm_arg);
+        if(alarm_cb) {
+            rtt_cb_t tmp = alarm_cb;
+            alarm_cb = NULL;
+            tmp(alarm_arg);
+        }
     }
     if (RTT_DEV->CRL & RTC_CRL_OWF) {
         RTT_DEV->CRL &= ~(RTC_CRL_OWF);
-        overflow_cb(overflow_arg);
+        if(overflow_cb) {
+            rtt_cb_t tmp = overflow_cb;
+            overflow_cb = NULL;
+            tmp(overflow_arg);
+        }
```

Comment:
Do we really want to clear the overflow callback?

Reply:
Maybe not? Kind of based myself on the stm32/rtt_all code for this one change TBH

Reply:
Should I remove that for the alarm case as well?

Reply:
Ah I see that the interrupt gets disabled as well.
Then it doesn't matter if the callback is cleared or not. It would get overwritten when re-enabling the interrupt.

But I find it odd that the overflow callback is only triggered once.
Unfortunately the API is not defining whether the callback should be called on *every* overflow or only on the *next* overflow.

Reply:
> But I find it odd that the overflow callback is only triggered once.

Maybe, but lets keep the same behavior as for `alarm_cb` for now. I amended the change.